packages/sanity/src/core/config/document/fieldActions/define.ts

```diff
@@ -0,0 +1,8 @@
+import {DocumentFieldAction} from './types'
+
+/** @internal */
+export function unstable_defineDocumentFieldAction(
```

Comment:
Unsure if unstable_ prefix on a passthrough typehelper helper is necessary? It is marked `@internal` afterall, and the API is unstable_ already.

Reply:
Yes, good point, will remove the `unstable_` here.